runners/wikiConsole/templates/buildJava.asciidoc

```diff
@@ -0,0 +1,19 @@
+Build a Java Project with Maven
```

Comment:
Better write 'Build the Java Project with Maven' than 'Build a ...', because the output will refer to a specific project